addons/hr_contract/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,25 @@
+---Fields in module 'hr_contract'---
+hr_contract  / hr.contract              / active (boolean)              : NEW 
+# DONE: post-migration: filled as True by default
+
+hr_contract  / hr.contract              / activity_ids (one2many)       : NEW relation: mail.activity
+hr_contract  / hr.contract              / message_last_post (datetime)  : DEL 
+hr_contract  / hr.contract              / message_main_attachment_id (many2one): NEW relation: ir.attachment
+# NOTHING TO DO: related to mail mixins
+
+hr_contract  / hr.contract              / reported_to_secretariat (boolean): NEW
+# NOTHING TO DO: new feature
+
+hr_contract  / hr.employee              / children (integer)            : module is now 'hr' ('hr_contract')
+hr_contract  / hr.employee              / place_of_birth (char)         : module is now 'hr' ('hr_contract')
+# NOTHING TO DO: hr is a dependency of hr_contract
+
+hr_contract  / hr.employee              / vehicle_distance (integer)    : DEL
+# NOTHING TO DO: renamed to km_home_work in 'hr' module
+
+---XML records in module 'hr_contract'---
+DEL ir.model.access: hr_contract.access_hr_contract_type_user
+DEL ir.model.access: hr_contract.access_hr_contract_user
+NEW res.groups: hr_contract.group_hr_contract_manager (noupdate)
```

Comment:
As this new group now restricts some operations, we should make equivalent previous v11 permissions with these ones. As I'm seeing, this means that we should add this group to those users that are HR managers (`hr.group_hr_manager`).

Reply:
As a side note, I think this shouldn't be noupdate, but that's an Odoo bug.